lib/sass_spec/stats/todo.rb

```diff
@@ -0,0 +1,240 @@
+    regexes = []
+    if implementation == "dart-sass"
+      dart_sass = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- sass\/dart-sass#\d+$', Regexp::MULTILINE)
+      dart_sass_missing_issue = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- dart-sass$', Regexp::MULTILINE)
+      regexes.push(dart_sass, dart_sass_missing_issue)
+    elsif implementation == "libsass"
+      libsass = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- sass\/libsass#\d+$', Regexp::MULTILINE)
+      libsass_missing_issue = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- libsass$', Regexp::MULTILINE)
+      regexes.push(libsass, libsass_missing_issue)
+    end
+
+    todos = find_todos(files, regexes)
+    print_report(todos)
+  end
+
+  def todos_by_issue(files, issue)
+    if issue =~ /dart-sass/
+      regex = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- sass\/' + issue + '\d+$', Regexp::MULTILINE)
+    elsif issue =~ /libsass/
+      regex = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- sass\/' + issue + '$', Regexp::MULTILINE)
+    end
+
+    todos = find_todos(files, [regex])
+    print_report(todos)
+  end
+
+  def todos_missing_issue(files)
+    dart_sass = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- dart-sass$', Regexp::MULTILINE)
+    libsass = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- libsass$', Regexp::MULTILINE)
+
+    todos = find_todos(files, [dart_sass, libsass])
+    print_report(todos)
+  end
+
+  def todos(files)
+    dart_sass = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- sass\/dart-sass#\d+$', Regexp::MULTILINE)
+    dart_sass_missing_issue = Regexp.new(':todo:(?:\n- libsass)*?(?:\n- sass\/libsass)*?\n- dart-sass$', Regexp::MULTILINE)
+    libsass = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- sass\/libsass#\d+$', Regexp::MULTILINE)
+    libsass_missing_issue = Regexp.new(':todo:(?:\n- dart-sass)*?(?:\n- sass\/dart-sass)*?\n- libsass$', Regexp::MULTILINE)
+
+    todos = find_todos(files, [
+      dart_sass,
+      dart_sass_missing_issue,
+      libsass,
+      libsass_missing_issue,
+    ])
+    print_report(todos)
+  end
+
+  def find_todos(files, regexes)
+    todos = {}
+
+    files.each do |file|
+      next unless [".hrx", ".yml"].include? File.extname(file)
+      content = File.read(file)
+      matches = regexes.flat_map do |regex|
+        content.scan(regex)
+      end
+
+      next unless matches.length > 0
+      path = Pathname.new(file).relative_path_from(Pathname.new(SassSpec::SPEC_DIR)).to_s
+      todos[path] = []
+
+      matches.uniq.each do |match|
+        entry = {}
+        if match =~ /dart-sass/
+          impl = "dart-sass"
+        elsif match =~ /libsass/
+          impl = "libsass"
+        end
+        entry[:impl] = impl
+        entry[:issue] = match =~ /#/ ? match.split("#")[-1] : "?"
+        todos[path].push(entry)
```

Comment:
Not necessarily something that needs to be fixed in this PR, but something that will probably need to be fixed at some point, is changing this to count the number of *specs* that are marked as TODO rather than counting the number of *options files*. A single options file at the root of a directory applies to every spec beneath that directory, so eventually you'll probably need to search for all tests (identified by `input.{scss,css,sass}` files) within the directory that contains the `options.yml`.